etcdctl/README.md

```diff
@@ -1063,8 +1073,8 @@ etcdctl --user=root:123 user passwd myuser
 #### Examples
 
 ``` bash
-etcdctl --user=root:123 user grant-role userA roleA
-# Role roleA is granted to user userA
```

Comment:
have spaces between the `#` and the output? `#abc` looks more cluttered than `# abc`